Packs/GmailSingleUser/Classifiers/classifier-gmail-single-user.json

```diff
@@ -1,160 +1,16 @@
 {
-	"id": "1c53b98e-c405-4025-8703-a5455f2ff140",
+    "id": "Gmail Single User",
+    "name": "Gmail Single User - Classifier",
+    "type": "classification",
+    "description": "Classifies Gmail phishing email messages.",
 	"version": -1,
-	"modified": "2019-11-26T15:49:02.222234104+03:00",
 	"defaultIncidentType": "",
```

Comment:
Shouldnt `defaultIncidentType` here be phishing ?
And add this incident type as a dependency for the pack